students/charlie_england/lesson01/assignment/inventory_management/electricAppliancesClass.py

```diff
@@ -0,0 +1,22 @@
+# Electric appliances class
+from inventoryClass import inventory
```

Comment:
Since the files are mainly going to be running in the assignment directory, it is a good idea to modify your import to not be relative:
from inventory_management.inventoryClass import inventory